chain/actors/actors.go

```diff
@@ -15,20 +15,18 @@ import (
 
 type Version int
 
-var LatestVersion = 8
-
-var Versions = []int{0, 2, 3, 4, 5, 6, 7, 8}
-
 const (
-	Version0 Version = 0
-	Version2 Version = 2
-	Version3 Version = 3
-	Version4 Version = 4
-	Version5 Version = 5
-	Version6 Version = 6
-	Version7 Version = 7
-	Version8 Version = 8
-	Version9 Version = 9
+	Version0  Version = 0
+	Version2  Version = 2
+	Version3  Version = 3
+	Version4  Version = 4
+	Version5  Version = 5
+	Version6  Version = 6
+	Version7  Version = 7
+	Version8  Version = 8
+	Version9  Version = 9
+	Version10 Version = 10
+	Version11 Version = 11
 )
 const (
 	AccountKey  = "account"
```

Comment:
will need to add the [evm actor key from fip-0054](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0054.md) here?